test/beast/core/basic_stream.cpp

```diff
@@ -83,11 +83,13 @@ class test_executor
     std::shared_ptr<state> sp_;
 
 public:
-    test_executor(test_executor const&) = default;
-    test_executor& operator=(test_executor const&) = default;
+    test_executor(test_executor const &) = default;
+
+    test_executor &
+    operator=(test_executor const &) = default;
```

Comment:
Why the gratuituous changes in whitespace? They just create noise in the diff...